src/tlo/methods/rti.py

```diff
@@ -3845,27 +3954,28 @@ def apply(self, person_id, squeeze_factor):
                 get_item_code('Blood, one unit'): 2,
                 get_item_code("Oxygen, 1000 liters, primarily with oxygen cylinders"): 23_040
             }
-            is_cons_available = self.get_consumables(
-                self.module.item_codes_for_consumables_required['shock_treatment_child']
-            )
+            # cons_available = self.get_consumables(item_codes=self.module.cons_item_codes['shock_treatment_child'])
+            cons_available = self.get_consumables(item_codes=
+                  self.module.item_codes_for_consumables_required['shock_treatment_child'])
```

Comment:
Not sure of the intention of these changes. I think the original is fine, isn't it?

Having read through the whole file, I am wondering if you're proposing this change in the name of the variable (from `is_cons_available` to `cons_available`) , because "is_X_available" sounds like a function that is testing something, whereas "X_available" sounds like the answer (a bool).

Is that right?

If so, then I guess I do agree with that in principle, but I also don't think we have to for this PR, that is focussed on something else, and I think it might confuse others / ourselves in the future, to make that incidental change throughout.